cluster/gce/util.sh

```diff
@@ -1362,7 +1362,7 @@ function kube-down() {
     "${NETWORK}-default-ssh" \
     "${NETWORK}-default-internal"  # Pre-1.5 clusters
   if [[ "${KUBE_DELETE_NETWORK}" == "true" ]]; then
-    delete-network
+    delete-network || true  # might fail if there are leaked firewall rules
```

Comment:
I should probably update the comment, since leaked VMs, routes, and other things will also prevent network deletion.

Basically, with the `gcloud` API it'll only delete the network if there are no resources in it. In the cloud console (web) it'll first delete anything in the network.